src/Elmish.WPF/ViewModel.fs

```diff
@@ -9,63 +9,95 @@ open System.Windows
 
 open Elmish
 
+
+type internal OneWay<'model, 'a> = {
```

Comment:
We could suffix the type names with `Data`, just like we do in `Binding.fs`. What do you think?

Reply:
I think adding a suffix is a good idea.  I would prefer to keep the names distinct, so I don't want to also suffix these ones with `Data`.  Those records in `Bindings.fs` are the types for cases in the DU called `BindingData`:
https://github.com/elmish/Elmish.WPF/blob/12ad75bd2d82398b11f5b4b7d0d60eb2909921be/src/Elmish.WPF/Binding.fs#L111-L122

How about we suffix the new records with `Binding` since they are the types for cases in the DU called `VmBinding`?
https://github.com/elmish/Elmish.WPF/blob/12ad75bd2d82398b11f5b4b7d0d60eb2909921be/src/Elmish.WPF/ViewModel.fs#L13

Reply:
👍

Reply:
Done